docset/winserver2022-ps/pki/Switch-Certificate.md

```diff
@@ -11,31 +11,42 @@ title: Switch-Certificate
 # Switch-Certificate
 
 ## SYNOPSIS
+
 Marks one certificate as having been replaced by another certificate.
 
 ## SYNTAX
 
 ```
-Switch-Certificate [-NotifyOnly] [-NewCert] <Certificate> [-OldCert] <Certificate> [-WhatIf] [-Confirm]
- [<CommonParameters>]
+Switch-Certificate [-NotifyOnly] [-NewCert] <Certificate> [-OldCert] <Certificate> [-WhatIf]
+ [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Switch-Certificate** cmdlet marks one certificate as having been replaced by another certificate.
-This cmdlet triggers a replace certificate notification and optionally sets the renewal property on the certificate being replaced.
+
+The `Switch-Certificate` cmdlet marks one certificate as having been replaced by another
+certificate. This cmdlet triggers a replace certificate notification and optionally sets the renewal
+property on the certificate being replaced.
 
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
-PS C:\>Switch-Certificate -OldCert cert:\LocalMachine\My\E42DBC3B3F2771990A9B3E35D0C3C422779DACD7 -NewCert cert:\LocalMachine\My\4A346B4385F139CA843912D358D765AB8DEE9FD4
+
+```powershell
+$params = @{
+    OldCert = 'Cert:\LocalMachine\My\E42DBC3B3F2771990A9B3E35D0C3C422779DACD7'
+    NewCert = 'Cert:\LocalMachine\My\4A346B4385F139CA843912D358D765AB8DEE9FD4'
+}
+Switch-Certificate @params
 ```
 
-This example sets the renewal property of the certificate with the thumbprint E42DBC3B3F2771990A9B3E35D0C3C422779DACD7 as renewed by the certificate with the thumbprint 4A346B4385F139CA843912D358D765AB8DEE9FD4 and generates a replace certificate notification.
+This example sets the renewal property of the certificate with the thumbprint
+E42DBC3B3F2771990A9B3E35D0C3C422779DACD7 as renewed by the certificate with the thumbprint
+4A346B4385F139CA843912D358D765AB8DEE9FD4 and generates a replace certificate notification.
 
 ### EXAMPLE 2
-```
-PS C:\>Set-Location -Path cert:\LocalMachine\My
+
+```powershell
+Set-Location -Path cert:\LocalMachine\My
 
 PS cert:\LocalMachine\My>$oldCert = Get-ChildItem -Path E42DBC3B3F2771990A9B3E35D0C3C422779DACD7
```

Comment:
I can't comment on the line after this one, but we should remove the prompt from all lines in this example and remove the extra newlines.

Reply:
Whoops. I misread this: https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.3#avoid-using-powershell-prompts-in-examples

I'll fix these along with the Synopsis fix..